packages/http/src/models/Sum.ts

```diff
@@ -0,0 +1,52 @@
+import Balance, { CurrencyType, NetworkTokens } from '@helium/currency'
+import DataModel from './DataModel'
+import Client from '../Client'
+
+export type RewardSumData = Omit<Sum, 'client'>
+
+export interface HTTPSum {
+  total: number
+  sum: number
+  stddev: number
+  min: number
+  median: number
+  max: number
+  avg: number
+}
+
+function floatToBalance(floatValue: number): Balance<NetworkTokens> {
+  return Balance.fromFloat(floatValue, CurrencyType.networkToken)
```

Comment:
Can you add a check here

```
if (!number) return undefined
```

I think challenge sums dont have a total so we should leave that out for those ones

Reply:
I started doing that, but figured it would be out of the scope of this PR and could be handled in the challenge PR

Reply:
Ok sounds good